src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt

```diff
@@ -348,7 +410,22 @@ private fun onWebsocketReady() {
         }
 }
 
-private fun checkEvents(target: EventTarget?, typesToCheck: Collection<EventType>) {
+private fun checkHoverEvents(target: EventTarget?, hoverTooltip: HTMLDivElement) {
+    if (target is HTMLSpanElement && target.classList.contains(COMPONENT_CLASS)) {
```

Comment:
I don't think this needs the same treatment with the collection of event types. This method can just stay as is and we can remove hover from that enum.

Reply:
It doesn't have a collection of event types, i just renamed the old `checkEvents` method as to not cause confusion between click and hover events, but this hover event method only existing because it recurses